src/sentry/integrations/gitlab/webhooks.py

```diff
@@ -63,17 +57,11 @@ def update_repo_data(self, repo, event):
 
         project = event["project"]
 
-        name_from_event = "{} / {}".format(project["namespace"], project["name"])
```

Comment:
If we really did want to update the name, we could do it with some fancy regex, but I opted to remove it since we have a constraint on the DB that enforces unique names. I'd rather have the behavior be consistent across the board than only work if the name passes the constraint.